src/time.cc

```diff
@@ -2,109 +2,117 @@
 #include <stdint.h>
 #include <time.h>
 
-#include "nan.h"
+#include "napi.h"
+#include "uv.h"
```

Comment:
Is this used?

Reply:
nope 9b6f5f21fccd6b4a432f9fb0ba4b6750436dc49f